backend/src/main/java/de/robinschatzl/gameapp/backend/game/GameService.java

```diff
@@ -18,4 +18,8 @@ public List<Game> getAllGames() {
     public Game addGame(Game gameToAdd) {
         return gameRepoInterface.save(gameToAdd);
     }
+
+    public Game getGameById(String id) {
+        return gameRepoInterface.findById(id).orElseThrow();
```

Comment:
Schön wäre es hier auch tatsächlich eine Exception zu werfen z.B. NoSuchElementException
```suggestion
        return gameRepoInterface.findById(id).orElseThrow(() -> new NoSuchElementException("Game with id \"" + id + "\" not found ));
```

Reply:
stimmt würde sinn machen